### DISCUSSION TITLE
how can I enable entity resolution

### DISCUSSION THREAD

**Discussion by AlricLou** [microsoft/graphrag#778] [Q&A]
I can't find anywhere using GRAPHRAG_ENTITY_RESOLUTION_ENABLED, and I set GRAPHRAG_ENTITY_RESOLUTION_ENABLED=True in .env file, it seems doesn't work, still produce duplicate entities

**Comment by natoverse** [comment]
It is not currently implemented, the setting is left over from an entity resolution step we had in the codebase but were not happy with. The docs are being updated now to remove that config.

**Reply by Tidomat** [reply]
Will there be an entity resolution step added in the future, or would you recommend we work on implementing this ourselves? Thanks 🙏🏻

**Reply by natoverse** [reply]
We are researching better entity resolution approaches, but don't have any planned implementation date.

**Comment by guangxiangdebizi** [comment]
plus one
